deskpro/deskpro-admin-frontend
# Feature/5197 actions no endpoints rebase

To test:

Go to agents table page
Check a row in the table
Click actions
Click menu item No side effect action
See console log
